libpod/options.go

```diff
@@ -345,7 +346,7 @@ func WithCNIConfigDir(dir string) RuntimeOption {
 			return define.ErrRuntimeFinalized
 		}
 
-		rt.config.CNIConfigDir = dir
+		rt.config.NetworkDir = dir
```

Comment:
Really dislike the name here - this doesn't really tell me much. `NetworkConfigDir` maybe?